modules/biobank/sql/biobankSchema.sql

```diff
@@ -0,0 +1,466 @@
+-- DROPS --
```

Comment:
I think the correct place for this file is in the install schema or in `SQL/New_patches`. It's not standard for modules to have a `sql/` file within them.